flakeheaven/_logic/_extractors.py

```diff
@@ -220,10 +221,13 @@ def extract_flake8_pytest_style() -> Dict[str, str]:
 
 
 def extract_flake8_annotations_complexity() -> Dict[str, str]:
-    from flake8_annotations_complexity.checker import AnnotationsComplexityChecker
```

Comment:
is this class/module not available anymore? or what is the reason for this change?

Reply:
The plugin code had changed so the existing test was failing. The class exists:
https://github.com/best-doctor/flake8-annotations-complexity/blob/master/flake8_annotations_complexity/checker.py

However AnnotationsComplexityChecker._error_message_template no longer exists.